.eslintrc

```diff
@@ -11,17 +11,22 @@
     "expect": true,
     "sinon": true
   },
+  "extends": ["eslint:recommended", "prettier"],
+  "plugins": ["prettier"],
   "rules": {
+    "prettier/prettier": [
+      "error",
+      {
+        "printWidth": 100,
+        "singleQuote": true, 
+        "trailingComma": "es5"
```

Comment:
Please, no. Don't force trailing commas.
1. It doesn't make any change in code quality, especialy when we have linter and precommit hook and we are abble to fix all the issues before the commit.
2. As far as I know we are pre ES5 compliant.
3. It's just the lazy programmer's excuse. "Hey, look. If I would like to add another array element, the comma is already there".

Reply:
Looking at our browser compatibility table ES5 looks like a requirement. I prefer trailing comma since I can work faster with it and have to think less. It can also reduce noise in commits by not changing two lines when adding at the end. @lmccart what's your take on it? Also see if there are other style rules to change.

Reply:
> I prefer trailing comma since I can work faster with it and have to think less.

Frankly speaking this can be stated about almost every code formatting rule ;)
All in all it's a matter of personall feelings.

All I want to say is, we should chose a subset od settings, that will be widely acceptable (it's hard).
Trailing commas is not so widely acceptable rule (basing on my experience with various open source and commercial projects).

Reply:
Not sure if constructive as I don't have any opinion on this, I normally omit trailing commas just out of habit. May be a bit obstructive to force it though but I don't expect it to be too big a problem for me.

Reply:
> All in all it's a matter of personall feelings.
Yes, exactly. There is an argument to be made that we should stick to the defaults of prettier, meaning no trailing comma. Any other rules to remove / add?

Reply:
Just like you, I also prefer trailing commas for speed,  but p5 has been using idiomatic.js. which does not use them,  so when I'm workyon p5 I leave them out.   I think it's better to keep the eslint and prettier as close to idiomatic.js rules as possible since that's been the standard for p5.

Reply:
it's a matter of personal preference of course. I would agree with @Jared-Sprague here, omitting the trailing commas also aligns with the documentation on the rest of the website so I think it would be more intuitive for beginner developers coming to the codebase which is a good guiding principle in terms of the rules in general I think.

Reply:
i vote for trailing commas. it makes code motion significantly simpler.